libs/datasets/combined_datasets.py

```diff
@@ -33,7 +33,6 @@
 from libs.datasets.sources.can_scraper_local_dashboard_providers import CANScraperStateProviders
 from libs.datasets.sources.can_scraper_usafacts import CANScraperUSAFactsProvider
 from libs.datasets.sources.cdc_testing_dataset import CDCTestingDataset
-from libs.datasets.sources.covid_care_map import CovidCareMapBeds
```

Comment:
🎉

Reply:
Note for posterity. We originally used Covid Care Map as a static source of 2019 bed capacities. Dave Luo was very helpful.

However, they started updating their data by mixing in updated values as they found them through state resources and news articles. This hybrid datasource was not viable for us since we made some corrections and estimates with the assumption that these were 2019 numbers, not nowcasting numbers.

In the end we ended up switching over the HHS Facilities Level Hospital Dataset before we resolved this issue with them.